src/test/java/com/epam/tc/hw2/ex1/HomePageTest.java

```diff
@@ -0,0 +1,83 @@
+package com.epam.tc.hw2.ex1;
+
+import io.github.bonigarcia.wdm.WebDriverManager;
+import java.util.List;
+import java.util.concurrent.TimeUnit;
+import org.assertj.core.api.SoftAssertions;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.testng.annotations.AfterMethod;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+
+@Test
+public class HomePageTest {
+
+    private WebDriver driver;
+
+    @BeforeMethod
+    public void setupDriver() {
+        WebDriverManager.chromedriver().setup();
+    }
+
+    @Test
+    public void testHomePage() {
+        driver = new ChromeDriver();
+        SoftAssertions softAssertions = new SoftAssertions();
+        driver.manage().window().maximize();
+        driver.manage().timeouts().implicitlyWait(4, TimeUnit.SECONDS);
+        String url = "https://jdi-testing.github.io/jdi-light/index.html";
+        driver.navigate().to(url);
+        softAssertions.assertThat(driver.getCurrentUrl()).isEqualTo(url);
+        softAssertions.assertThat(driver.getTitle()).isEqualTo("Home Page");
+        driver.findElement(By.xpath("/html/body/header/div/nav/ul[2]/li/a")).click();
+        driver.findElement(By.xpath("//*[@id=\"name\"]")).sendKeys("Roman");
+        driver.findElement(By.id("password")).sendKeys("Jdi1234");
+        driver.findElement(By.id("login-button")).click();
+        WebElement webElement = driver.findElement(By.xpath("//*[@id='user-name']"));
+        softAssertions.assertThat(webElement.getText()).isEqualTo("ROMAN IOVLEV");
+        webElement = driver.findElement(By.xpath("/html/body/header/div/nav/ul[1]/li[1]/a"));
+        softAssertions.assertThat(webElement.getText()).isEqualTo("HOME");
+        webElement = driver.findElement(By.xpath("/html/body/header/div/nav/ul[1]/li[2]/a"));
+        softAssertions.assertThat(webElement.getText()).isEqualTo("CONTACT FORM");
+        webElement = driver.findElement(By.xpath("/html/body/header/div/nav/ul[1]/li[3]/a"));
+        softAssertions.assertThat(webElement.getText()).isEqualTo("SERVICE");
+        webElement = driver.findElement(By.xpath("/html/body/header/div/nav/ul[1]/li[4]/a"));
+        softAssertions.assertThat(webElement.getText()).isEqualTo("METALS & COLORS");
```

Comment:
Please use one css locator here that selects all required items

Reply:
corrected